main.js

```diff
@@ -29,6 +29,9 @@ function main(args) {
         .usage('[options]')
         .option('-l, --listen <listen>', 'Host and port on which server listens (also node\'s identity in cluster)')
         .option('-h, --hosts <hosts>', 'Seed file of list of hosts to join')
+        .option('--suspect [suspect]', 'Suspect period in seconds')
```

Comment:
I think these should be `suspect-period`, `faulty-period` and `tombstone-period` respectively, since, at least I am used to noun being a boolean (binary) meaning.